core/src/main/java/org/bitcoinj/params/MainNetParams.java

```diff
@@ -348,16 +348,6 @@ public static synchronized MainNetParams get() {
         return instance;
     }
 
-    // TODO: Until 19.2 is released on mainnet, we must use 70227 as the min and current version
-    @Override
-    public int getProtocolVersionNum(ProtocolVersion version) {
-        if (version == ProtocolVersion.CURRENT)
-            return ProtocolVersion.CURRENT.getBitcoinProtocolVersion() - 1;
-        else if (version == ProtocolVersion.MINIMUM)
-            return ProtocolVersion.MINIMUM.getBitcoinProtocolVersion() - 1;
-        return super.getProtocolVersionNum(version);
-    }
```

Comment:
Since we will only connect to 19.2 nodes, we don't need to worry about 19.2 nodes that use a lower protocol version.